### DISCUSSION TITLE
Changing color of Bookmark Icons in left-sidebar to match theme

### DISCUSSION THREAD

**Discussion by eagrunewald** [Godiesc/firefox-gx#144] [General]
Hi, first of all thanks for this beautiful theme to firefox, i loved it. I would like to know how did you make your icons on the left sidebar change colors to match the theme color, like show in your first preview gif and how to remove the new tab image from left sidebar and tabs backgrounds but keeping it in new tab background like in privacy new window. Thanks in advance.

**Comment by Tintouin** [comment]
Well, I have this result and the things I did were here : https://github.com/Godiesc/firefox-gx/blob/main/Extras/Left-SideBar/README.md
Basically installing the right extensions will add a name that match with some names in the icon folder.

**Reply by eagrunewald** [reply]
My folders icon change colors like the preview gif, but my Skype, Whattsapp, YouTube icons don't. In the preview gif all icons changed colors and my question is how to achieve this result.

**Comment by Godiesc** [comment]
hi,
the icons that adapt its color are: default firefox icons, bookmark-folders and some extensions listed in the redme of left-sidebar:

![imagen](https://github.com/Godiesc/firefox-gx/assets/22057609/3a8d7890-17bb-4c97-92aa-31a3c715801b)

for bookmark sites the favicon is the icon showed, there is no simple way to change that, you should change manually for each one, you can see the bug here: https://github.com/Godiesc/firefox-gx/issues/128

**Reply by eagrunewald** [reply]
Many thanks! I followed the steps and everything is perfect now!
